frontend/src/main/pages/Admin/AdminJobLogPage.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import { Button } from "react-bootstrap";
+import { useBackend } from "main/utils/useBackend";
+
+const AdminJobLogPage = () => {
+  const { id } = useParams();
+  const navigate = useNavigate();
+
+  const { data: jobLogs } = useBackend([`/api/jobs/logs/${id}`], {
+    method: "GET",
+    url: `/api/jobs/logs/${id}`,
+  });
+
+  const handleBack = () => {
+    navigate("/admin/jobs");
+  };
+
+  return (
+    <div>
```

Comment:
Look at every single other Page component in the entire app.  What do you notice?

```suggestion
    <BasicLayout>
```